recipe_scrapers/_schemaorg.py

```diff
@@ -309,3 +309,12 @@ def cooking_method(self):
         if cooking_method and isinstance(cooking_method, list):
             cooking_method = cooking_method[0]
         return normalize_string(cooking_method)
+
+    def dietary_restrictions(self):
+        dietary_restrictions = self.data.get("suitableForDiet")
+        if dietary_restrictions is None:
+            raise SchemaOrgException("No dietary restrictions data in SchemaOrg.")
+
+        if isinstance(dietary_restrictions, list):
+            return ", ".join(format_diet_name(diet) for diet in dietary_restrictions)
```

Comment:
I'd probably suggest adding a split-by-commas step if the `dietary_restrictions` is a string before this, so that we can handle CSV-style entries too.

Reply:
Could implement `csv_to_tags` that will become available with #1128 

Is that what you had in mind?

Reply:
Exactly, yep - in combination with the diet name validation logic here too.

Reply:
should be all set!